src/components/AddProperty.js

```diff
@@ -0,0 +1,155 @@
+import React, { useState } from 'react';
+import axios from 'axios';
+import '../styles/AddProperty.css';
+
+function AddProperty() {
+  const initialState = {
+    fields: {
+      title: '',
+      city: 'Your City',
+      type: 'Type of Residence',
+      bedrooms: 'Number Of Bedrooms',
+      bathrooms: 'Number of bathrooms',
+      price: 0,
+      email: 'name@email.com',
+    },
+  };
+  const [fields, setFields] = useState(initialState.fields);
+
+  const handleAddProperty = (event) => {
+    event.preventDefault();
+    console.log(fields);
+    axios.post('http://localhost:3000/api/v1/PropertyListing', fields).then(
+      (res) => {
+        console.log(res.data);
```

Comment:
Is the console log providing any use to a user here? Would be better for users to display some message about the outcome of trying to add a property, theres a few ways to go about doing this but the simplest is probably to display a div on the page with a highlighted bit of text with a bit of info stating success, or if it failed, the same div could be used to hold a failure message created in the error block below